packages/lesswrong/lib/editor/ellipsize.jsx

```diff
@@ -61,6 +61,18 @@ export const getTruncationCharCount = (comment, currentUser, postPage) => {
   }
 }
 
+export const answerTocExcerptFromHTML = (html) => {
+  if(!html) return ""
+  const styles = html.match(/<style[\s\S]*?<\/style>/g) || ""
```

Comment:
If I'm reading correctly, this line is unused.

Reply:
I think you're reading it incorrectly. (But, also, I've changed it since you commented so not sure about the original state)

Reply:
I was indeed reading it incorrectly.